sdk/eventhub/event-hubs/src/models/private.ts

```diff
@@ -125,6 +125,10 @@ export interface EventHubConsumerOptions {
    * prefer to work directly with the bytes present in the message body than have the client attempt to parse it.
    */
   skipParsingBodyAsJson?: boolean;
+  /**
+   * The count of events requested eagerly and queued without regard to whether a read was requested.
+   */
+  PrefetchCount?: number;
```

Comment:
why caps? why not "prefetchCount"?

Reply:
Thanks! it was a typo, addressed in https://github.com/Azure/azure-sdk-for-js/pull/26065/commits/9314b6a039f57fbde2947ac4910ddf255974880a